src/ucp/core/ucp_request.h

```diff
@@ -60,6 +61,7 @@ typedef struct ucp_send_state {
  * Request in progress.
  */
 struct ucp_request {
+    ucs_status_t                  status;  /* Operation status */
```

Comment:
Do we need this field as the first one in the structure? It might displace usefully data from the first cache line.